test.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import unittest
+
+
+class TestServerFunctionality(unittest.TestCase):
+
+	def setUp(self):
+		self.url= 'http://ec2-52-11-248-121.us-west-2.compute.amazonaws.com/api/v1'
+		self.loginval= {"login":"test", "password":"test"}
+
+	def test_1Login(self):
```

Comment:
1. try to define test name more readable  test_login_positive
2. if you using _ in method name then all character of word should be with lover case word_word_word for tests we using it. Or camel case wordWordWord for ordinary methods.